### ISSUE TITLE
Docs: Getting started redownloads dataset instead of reading downloaded data set

### ISSUE THREAD

**Issue by rraihansaputra** [modin-project/modin#4836]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **Modin version** (`modin.__version__`):
- **Python version**:
- **Code we can use to reproduce**:

### Describe the problem
The Getting Started docs needs a correction to the `pd.read_csv` parameter to read dataset directly from disk to compare `pandas` and `modin.pandas` correctly.

Currently, each run will re-download the dataset and run the test, in which the difference in `read_csv` speeds is not reflected as the majority of the time is used to download the file.

### Source code / logs
https://github.com/modin-project/modin/blame/master/docs/getting_started/quickstart.rst#L83

Line 83 and line 96 uses the `s3_path` parameter for the file which is the url, instead of the path to the downloaded `taxi.csv` file.

I'm willing to create a PR for this.

Extra note: there's also a possibility of improvement by checking whether the file is already downloaded and avoid redownloading when running the dataset download cell.

**Comment by pyrito** [comment]
@rraihansaputra Thanks for opening the issue! After looking into it, you are correct that it's not showing the time spent reading it from disk but from the object store instead. I think it's still a valid comparison since reading from an object store is a normal use case. However, I'm not sure what the documentation was trying to show here, and I think we should have been more explicit. @modin-project/modin-contributors @modin-project/modin-core thoughts?

**Comment by mvashishtha** [comment]
> however, I'm not sure what the documentation was trying to show here

@pyrito the page is supposed to make Modin look faster :) We could download the dataset first as [in the quickstart](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html#example-instant-scalability-with-no-extra-effort). We could explain that 1) we are doing that to show the perf improvement 2) it's still possible to read directly from s3, and normally that's how one would read the data.

**Comment by pyrito** [comment]
Agree with the second point. It's still completely valid to read directly from s3 since that's a common use case and ends up being a bottleneck in terms of performance for many, so we are showing clear performance improvement there.

**Comment by rraihansaputra** [comment]
> @pyrito the page is supposed to make Modin look faster :) We could download the dataset first as [in the quickstart](https://modin.readthedocs.io/en/latest/getting_started/quickstart.html#example-instant-scalability-with-no-extra-effort). We could explain that 1) we are doing that to show the perf improvement 2) it's still possible to read directly from s3, and normally that's how one would read the data.

Yeah I'm referring to the same page. The bug is that the file is actually downloaded, but the next cells refers to the `s3_path` which is the object path, rather than referring to read the downloaded file.

I believe reading from local emphasizes the improvements from using modin. Reading from an object store is a valid usecase, but I think it can be demonstrated somewhere else, not on this page.

**Comment by devin-petersohn** [comment]
Reading from S3 should still be faster in Modin, and if it is not, that's a bug 😄

**Comment by pyrito** [comment]
That makes sense @rraihansaputra ! Do you want to go ahead and open a PR for this?
